src/rust/ide/view/src/documentation.rs

```diff
@@ -97,8 +81,8 @@ impl Model {
 
         dom.dom().set_attribute_or_warn("class"       ,"scrollable"                 ,&logger);
         dom.dom().set_style_or_warn("white-space"     ,"normal"                     ,&logger);
-        dom.dom().set_style_or_warn("overflow-y"      ,"auto"                       ,&logger);
-        dom.dom().set_style_or_warn("overflow-x"      ,"auto"                       ,&logger);
+        dom.dom().set_style_or_warn("overflow-y"       ,"auto"                       ,&logger);
+        dom.dom().set_style_or_warn("overflow-x"       ,"auto"                       ,&logger);
```

Comment:
This change seems to break the alignment (and do nothing else).

Reply:
That's strange because it's perfectly aligned in my IDE:
<img width="655" alt="Zrzut ekranu 2021-08-13 o 15 55 11" src="https://user-images.githubusercontent.com/26655166/129367897-3184da5c-7ef5-49f5-a73e-f3ba1fa67a13.png">